src/blockchain/milestones.js

```diff
@@ -248,23 +335,29 @@ const milestonesFactory = app => {
         );
         return;
       }
-
-      // if (!milestone.maxAmount || !milestone.fullyFunded) return;
-      // never set uncapped or non-fullyFunded milestones as PAID
-      if (!matchingMilestones[0].maxAmount || !matchingMilestones[0].fullyFunded) return;
```

Comment:
This part should not be removed

Reply:
It wasn't removed already, it just moved to some lines later.


![image](https://user-images.githubusercontent.com/9850545/101512465-4758ae00-3990-11eb-97cd-24d25650777c.png)